### PR TITLE
Add django 110 support

### PR DESCRIPTION
**Pull request by rlconley**
This pr adds support for Django 1.10 in the project, example app, and tests

It also fixes the class FunctionalTestCase which was broken due to a Firefox/Selenium incompatibility, issue#130 and a problem of timing (looking for an element on the page that wasn't there), issue#129. I first tried to switch to Chrome, but that would require some very brittle code in the travis.yml file to put chromedriver in the right place. The tests didn't have css anyway, so I switched to PhantomJS.

### CODE REVIEW

File: scribbler/tests/urls.py [hunk 1]
```diff
@@ -12,6 +13,7 @@ def test_404(request):
 
 
 def test_500(request):
+    import pdb; pdb.set_trace()
```
Comment (victor-rocha) [inline scribbler/tests/urls.py]: Remove this line, please.

Comment (rlconley) [inline scribbler/tests/urls.py]: Oops. Thanks!

**Comment by victor-rocha** [comment]
:100:

**Comment by dpoirier** [comment]
@vrocha if this is okay to merge, can you click "Accept Review" on the pull request? Or let me know what the issues are and I'll work on them.

**Pull request merged**